nsis/src/nsis/format/NsisScriptHeader.java

```diff
@@ -11,35 +11,40 @@
 import nsis.file.NsisConstants;
 
 public class NsisScriptHeader implements StructConverter {
+	public final int flags;
+	private byte[] siginfo;
 	private byte[] magic;
 	public final int inflatedHeaderSize;
 	public final int archiveSize;
 	public final int compressedHeaderSize;
-	public final int flags;
 	private final static Structure STRUCTURE;
 
 	static {
 		STRUCTURE = new StructureDataType("script_header", 0);
-		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "magic", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "inflated_header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "compressed_header_size", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", null);
+		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", "First header flags (FH_FLAGS_*)");
+		STRUCTURE.add(STRING, NsisConstants.NSIS_SIGINFO.length, "siginfo", "0xdeadbeef (FH_SIG)");
+		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "nsinst",
+				"NSIS magic (FH_INT1, FH_INT2, FH_INT3)");
+		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size",
```

Comment:
right now these fields and their counterparts in NsisScriptHeader are out of sync naming-wise (length_of_following_data and archiveSize, header_size and inflatedHeaderSize). Please be consistent.

Reply:
length_of_following_data and header_size come from the NSIS file spec, archiveSize and inflatedHeaderSize are the names we gave them so its more clear what they mean. Should I rename them to their NSIS spec name?